resources/styles.css

```diff
@@ -211,3 +219,50 @@ input.tab-text {
     width: 20px;
     height: 20px;
 }
+
+
+
+
+html, body, .r_flex_container, #app{
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+}
+.r_flex_container {
+    display:flex;
+    flex-flow: column nowrap;
+}
+
+.r_flex_fixed_child_top {
+    background-color:black;
+    flex:none;
+    margin-bottom: 50px;
+}
+
+.r_flex_fixed_child_bott {
+    background-color:black;
+    flex:none;
+    margin-bottom: 0px;
+}
+
+.r_flex_expand_child {
+    flex:auto;
+    overflow-y:auto;
+}
+
+#left {
```

Comment:
left and right should **not** be ids.

css ids should be unique, this means that they should be avoided for things that we are not sure will be unique (like right elements)

Also, make sure there is not any unwanted collision with these, as left/right are extremely common css classes

Reply:
Ops, I did that just trying and forgot to change it! I'll do